booster-android-gradle-v3_4/src/main/kotlin/com/didiglobal/booster/android/gradle/v3_4/V34.kt

```diff
@@ -161,6 +162,9 @@ object V34 : AGPInterface {
     override val BaseVariant.mergedRes: Collection<File>
         get() = getFinalArtifactFiles(InternalArtifactType.MERGED_RES)
 
+    override val BaseVariant.mergedNativeLibs: Collection<File>
+        get() = setOf((File("${project.buildDir.path}/$FD_INTERMEDIATES/transforms/mergeJniLibs/$name")))
```

Comment:
Please use `File.pathSeparator` instead of `\`

Reply:
use File.separatorChar to replace "\",please review again